backend/clickhouse/query.go

```diff
@@ -238,121 +248,17 @@ func makeSelectBuilder[T ~string](config tableConfig[T], selectStr string, selec
 		}
 	}
 
-	appendWhereConditions(sb, config, params.Query)
-
-	return sb, nil
-}
-
-func appendWhereConditions[T ~string](sb *sqlbuilder.SelectBuilder, config tableConfig[T], query string) {
```

Comment:
make `AssignSearchFilters` accept a template `[T ~string]` and the `tableConfig` type a public `TableConfig`